### DISCUSSION TITLE
Moving to the Fleet API via MyTeslamate

### DISCUSSION THREAD

**Discussion by aaronactive** [teslamate-org/teslamate#4438] [Q&A]
Hi All,

I got the email last week about the old API which no longer works and went through the process of setting up Tesla developer app etc as instructed by MyTeslaMate

Is it expected that streaming resolution is now really bad? It works..barely..but I've noticed the data in teslamate only updates every few minutes now.

This is going connecting via the fleet page 
- TESLA_API_HOST=https://fleet-api.prd.na.vn.cloud.tesla.com
- TESLA_AUTH_HOST=https://auth.tesla.com
- TESLA_AUTH_PATH=/oauth2/v3
- TESLA_AUTH_CLIENT_ID=xxxxxx
and using

- TESLA_WSS_HOST=wss://streaming.myteslamate.com
- TESLA_WSS_USE_VIN=true

Wondering if people are seeing the same now? And if this is what to expect going forward? Or am I missing something..

EDIT:
Ok after looking at some of the more recent data entries it looks like it has been like that for awhile now............

**Comment by bodosom** [comment]
I imagine it depends on what is most important. I brought up a Teslamate instance at MyTeslamate and had them connect it to the new API so I could compare something to my existing instances.  Some "issues" like transitions to and from online/offline are minor but something like the map of a trip are now of little value since there are significantly fewer updates. However if I compare a trip on Teslafi which is using the streaming API the map updates are consistently every 60s and the map display looks much smoother.

So either Teslafi is using an alternative stream or they're better at interpolation.

Teslafi reports this data usage:
```
Tesla API Usage For Today:
Vehicle State: 630 (Unlimited)
Vehicle Data: 166 (200)
Vehicle Location: 0 (Combined with Vehicle Data)
Streaming Data: 166 (Unlimited)
Commands: 0 (50)
Wake Command: 0 (15)
Charging Commands: 0 (5)
```
